watertap/unit_models/translators/tests/test_translator_asm1_adm1.py

```diff
@@ -0,0 +1,448 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+Tests for Translator ADM1-ASM1 unit model.
+
+Verified against approximated results results from:
+
+Nopens, I., Batstone, D, Copp, J, Jeppsson, U. Rosen, C., Volcke, E., Alex, J.,
+and Vanrolleghem, P.
+water Research Vol. 43 pp.1913–1923.
```

Comment:
Is this the old message from ADM1-ASM1 translator validation, or is this the correct reference here for ASM1-ADM1 translator?

Reply:
correct reference for data for test but used as trends instead of specific data points